Makefile

```
@@ -36,7 +40,7 @@ build: lint $(BINDIR)/$(BINNAME)
 SRC := $(shell find . -type f -name '*.go' -print) go.mod go.sum
 
 $(BINDIR)/$(BINNAME): $(SRC)
-	GO111MODULE=on go build $(GOFLAGS) -tags '$(TAGS)' -ldflags '$(LDFLAGS)' -o '$(BINDIR)'/$(BINNAME) ./cmd/hypper
+	go build $(GOFLAGS) -tags '$(TAGS)' -ldflags '$(LDFLAGS)' -o '$(BINDIR)'/$(BINNAME)${GOEXE} ./cmd/hypper
```

```suggestion
	go build $(GOFLAGS) -tags '$(TAGS)' -ldflags '$(LDFLAGS)' -o '$(BINDIR)'/$(BINNAME) ./cmd/hypper
```
Seems that `${GOEXE}` is a leftover, `$(BINNAME)` will already contain the `.exe` at this point. If not, we should use `${GOEXE}` in the clean target too.

---

I have removed it, make on windows is a bit quirky and can not access the variables from `go env` for some reason. This was a leftover from an experiment.